src/chi/components/data-table/data-table.scss

```
@@ -0,0 +1,759 @@
+        &.icon-plus,
+        &.icon-minus {
+          color: $color-icon-primary;
+        }
+      }
+
+      &::before {
+        display: none;
+      }
+    }
+
+    .chi-button {
+      &.-flat {
+        &.-active,
+        &.-hover,
+        &:active,
+        &:hover {
+          background-color: transparent;
+          border-color: transparent;
+          box-shadow: none;
+        }
+      }
+    }
+
+    &.-expandable {
+      display: none;
+
+      @include respond-to(md) {
+        display: flex;
+        flex-basis: 3.5rem;
+      }
+    }
+
+    &.-selectable {
+      display: none;
+
+      @include respond-to(md) {
+        display: flex;
+        flex-basis: 3rem;
+
+        .chi-checkbox__label {
+          margin-right: 0;
+        }
+      }
+    }
+
+    & .-asociated {
+      &:first-child {
+        margin-right: 1rem;
+      }
+
+      &::before {
+        content: attr(data-label);
+        display: inline-block;
+        font-weight: $font-weight-semi-bold;
+        margin-right: 0.5rem;
+      }
+
+      @include respond-to(md) {
+        display: flex;
+        flex-basis: 0%;
+        flex-grow: 1;
+        flex-shrink: 1;
+
+        &:first-child {
+          margin-right: 0.5rem;
+        }
+
+        &::before {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &.-sorting {
+    .chi-data-table__head {
+      .chi-data-table__cell {
+        .chi-icon {
+          opacity: 0;
+        }
+
+        &:hover {
+          cursor: pointer;
+
+          .chi-icon {
+            color: $color-icon-base;
+            opacity: 1;
+          }
+        }
+
+        &.-active {
+          border-bottom-color: $color-border-primary;
+          color: $color-text-primary-alt;
+
+          .chi-icon {
+            color: $color-icon-primary;            
+            opacity: 1;
+          }
+
+          &.-descending {
+            .chi-icon {
+              transform: rotate(180deg);
+            }
+          }
+        }
+      }
+    }
+  }
+
+  &.-no-border {
+    .chi-data-table__head,
+    .chi-data-table__body {
+      .chi-data-table__cell {
+        @include respond-to(md) {
+          border: 0;
+        }
+      }
+    }
+  }
+
+  &.-striped {
+    .chi-data-table__body {
+      .chi-data-table__row {
+        &:nth-child(even) {
+          background-color: $color-background-muted-lighter;
+        }
+      }
+    }
+  }
+
+  &.-hover {
+    .chi-data-table__body {
+      .chi-data-table__row {
+        &:hover,
+        &.-hover {
+          @include respond-to(md) {
+            background-color: $color-background-base-dark;
+
+            &.-active,
+            &.-selected,
+            &.-expanded {
+              background-color: $color-background-primary-light;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  &.-bordered {
+    .chi-data-table__body {
+      .chi-data-table__row {
+        .chi-data-table__cell {
+          border: 0.0625rem solid $color-border-base-light;
```

Same inconsistency issue with border-color here.

```suggestion
          border: 0.0625rem solid $color-border-base-dark;
```